pkg/apis/elasticsearch/v1/types.go

```diff
@@ -1,7 +1,7 @@
-package v1alpha1
+package v1
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
is it ok to have an import for `v1` in an unrelated package `v1`?  CLO uses `corev1` in some places for this import

Reply:
as long as there is no conflict it doesnt matter.  compilation should catch any issues